include/common/arch/mips/asm/non-atomic.h

```diff
@@ -0,0 +1,109 @@
+#ifndef _ASM_GENERIC_BITOPS_NON_ATOMIC_H_
+#define _ASM_GENERIC_BITOPS_NON_ATOMIC_H_
+
+#include <asm/types.h>
+#include "common/arch/mips/asm/bitops.h"
+#include "common/arch/mips/asm/utils.h"
```

Comment:
I don't see this "non-atomic.h" header used anywhere.
Can it be dropped?

Reply:
Yes, It's really useless，I have dropped it.